__tests__/app.test.js

```diff
@@ -0,0 +1,61 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("app", () => {
+  describe("server errors", () => {
+    it("404 GET /api/topics - a path that doesn't exist but is valid format", () => {
+      return request(app)
+        .get("/api/chopics")
+        .expect(404)
+        .then((body) => {
+          const message = body.body.message;
+          expect(message).toBe("Path not found");
+        });
+    });
+  });
+  describe("/api", () => {
+    it('200 GET /api returns message, "all ok"', () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.message).toBe("all ok");
+        });
+    });
+  });
```

Comment:
This isn't a necessary endpoint, try to say on target and only code what's on your tickets.

Also, this endpoint "/api" is used in a later on ticket!